pkg/ipam/types/types.go

```diff
@@ -310,10 +311,10 @@ type Subnet struct {
 	Name string
 
 	// CIDR is the IPv4 CIDR associated with the subnet
-	CIDR *cidr.CIDR
+	CIDR *netip.Prefix
```

Comment:
Is there any reason this needs to remain a pointer? In other words, can't we use `CIDR netip.Prefix` here?

Reply:
Good point, updated both fields to be values instead of pointers 👍